phpcs.xml

```diff
@@ -5,9 +5,11 @@
     <file>.</file>
```

Comment:
Why did this whole file get renamed?

The use of `phpcs.xml.dist` as the distributed version, and adding `/phpcs.xml` to `.gitignore`, means that developers can override aspects of `phpcs.xml.dist` with their local-only version of `phpcs.xml`.

(Note, that as of PHPCS 3.1.0, you can now use `.phpcs.xml.dist` and `.phpcs.xml` (with leading dot).)

Reply:
@GaryJones Because we are debating integrating XMP's dev_lib suite into the repo, though I'm leaning more towards [GrumPHP](https://github.com/phpro/grumphp).  If we go with XWP's suite, it has `phpcs.xml` in it. We want ours to be the master file.

Reply:
@hellofromtonya we can choose the file name with the dev-lib by setting the `export WPCS_STANDARD={filename}.xml` in a `.dev-lib` file.

Reply:
@GaryJones @ThierryA Renamed it to `phpcs.xml.dist`.  Update coming....